src/test/java/uk/gov/companieshouse/api/strikeoffobjections/service/impl/ObjectionServiceTest.java

```diff
@@ -602,4 +605,21 @@ void deleteAttachmentHandleNullHttpStatusApiResponseOnDeleteAttachment() {
                 any());
     }
 
+    @Test
+    public void willCallFileTransferGatewayForDownload() throws IOException, ServiceException {
```

Comment:
Is it not calling the api? Not the gateway?

Reply:
I think this is ok, service calls fileTransferApiClient which returns it's own FileTransferApiClientResponse object

Reply:
ah, i think you meant the test title. Yes, we don't have a gateway we have the FileTransferApiClient so would be better to rename the title.

Reply:
Changed name.